cmd/prometheus/main.go

```diff
@@ -173,6 +174,9 @@ func main() {
 	a.Flag("rules.alert.for-grace-period", "Minimum duration between alert and restored 'for' state. This is maintained only for alerts with configured 'for' time greater than grace period.").
 		Default("10m").SetValue(&cfg.forGracePeriod)
 
+	a.Flag("rules.alert.resend-delay", "Minimum amount of time to wait before resending an alert to Alertmanager. Must be lower than resolve_timeout in Alertmanager").
```

Comment:
:+1:, thanks for mentioning the resolve_timeout.

Reply:
We really should change Prometheus to send the end time appropriately, resolve_timeout should only be a fallback.

Reply:
@brian-brazil Would you like that work done as part of this PR or a separate PR?

What would be the appropriate end time to send for an unresolved alert? Eval time + query staleness time?

Reply:
That'd be separate I'd say.

You'd want to leave a few eval intervals for safety, plus allow for this setting.

Reply:
Sounds good. I will plan to work on that after this PR is merged.